### PR TITLE
SpaceToDepth update spec

### PR DESCRIPTION
**Pull request by pelszkow**
### Details:
 - *SpaceToDepth update spec*

### Tickets:
 - *45758*

### CODE REVIEW

**Review by jdanieck — APPROVED** [review]

File: docs/ops/movement/SpaceToDepth_1.md [hunk 1]
```diff
@@ -28,32 +49,15 @@
 
 **Inputs**
 
-*   **1**: `data` - input tensor of any type with rank >= 3. Required.
+*   **1**: `data` - input tensor of type T with rank >= 3. **Required**.
```
Comment (ggalieroc) [inline docs/ops/movement/SpaceToDepth_1.md:52]: T ---> \*T\*

File: docs/ops/movement/SpaceToDepth_1.md [hunk 2]
```diff
@@ -28,32 +49,15 @@
...
-**Detailed description**
-
-*SpaceToDepth* operation permutes element from the input tensor with shape `[N, C, D1, D2, ..., DK]`, to the output tensor where values from the input spatial dimensions `D1, D2, ..., DK` are moved to the new depth dimension. Refer to the [ONNX* specification](https://github.com/onnx/onnx/blob/master/docs/Operators.md#SpaceToDepth) for an example of the 4D input tensor case.
-
-The operation is equivalent to the following transformation of the input tensor `data` with `K` spatial dimensions of shape `[N, C, D1, D2, ..., DK]` to *Y* output tensor. If `mode = blocks_first`:
-
-    x' = reshape(data, [N, C, D1/block_size, block_size, D2/block_size, block_size, ... , DK/block_size, block_size])
-
-    x'' = transpose(x',  [0,  3, 5, ..., K + (K + 1), 1,  2, 4, ..., K + K])
+*   **1**: permuted tensor of type T and shape `[N, C * (block_size ^ K), D1 / block_size, D2 / block_size, ..., DK / block_size]`.
```
Comment (ggalieroc) [inline docs/ops/movement/SpaceToDepth_1.md:56]: T ---> \*T\*

Comment (pelszkow) [reply docs/ops/movement/SpaceToDepth_1.md]: done

File: docs/ops/movement/SpaceToDepth_1.md (hunk 1 above)
Comment (pelszkow) [reply docs/ops/movement/SpaceToDepth_1.md]: done

**Review by sdurawa — APPROVED** [review]

**Review by ggalieroc — APPROVED** [review]

**Comment by pelszkow** [comment]
@lazarevevgeny, can you look on SpaceToDepth spec and merge it if is OK.

File: docs/ops/movement/SpaceToDepth_1.md [hunk 3]
```diff
@@ -6,11 +6,32 @@
...
+
+    x' = reshape(data, [N, C, D1/block_size, block_size, D2/block_size, block_size, ... , DK/block_size, block_size])
+
+    x'' = transpose(x',  [0,  3, 5, ..., K + (K + 1), 1,  2, 4, ..., K + K])
+
+    y = reshape(x'', [N, C * (block_size ^ K), D1 / block_size, D2 / block_size, ... , DK / block_size])
+
+If `mode = depth_first`:
+
+    x' = reshape(data, [N, C, D1/block_size, block_size, D2/block_size, block_size, ..., DK/block_size, block_size])
```
Comment (lazarevevgeny) [inline docs/ops/movement/SpaceToDepth_1.md:24]: add spaces around `/`

File: docs/ops/movement/SpaceToDepth_1.md [hunk 4]
```diff
@@ -6,11 +6,32 @@
...
 **Short description**: *SpaceToDepth* operation rearranges data from the spatial dimensions of the input tensor into depth dimension of the output tensor.
 
+
+**Detailed description**
+
+*SpaceToDepth* operation permutes element from the input tensor with shape `[N, C, D1, D2, ..., DK]`, to the output tensor where values from the input spatial dimensions `D1, D2, ..., DK` are moved to the new depth dimension.
+
+The operation is equivalent to the following transformation of the input tensor `data` with `K` spatial dimensions of shape `[N, C, D1, D2, ..., DK]` to *Y* output tensor. If `mode = blocks_first`:
+
+    x' = reshape(data, [N, C, D1/block_size, block_size, D2/block_size, block_size, ... , DK/block_size, block_size])
```
Comment (lazarevevgeny) [inline docs/ops/movement/SpaceToDepth_1.md:16]: Add spaces around '/'

**Review by lazarevevgeny — APPROVED** [review]
Only minor comments. Approve in advance.

File: docs/ops/movement/SpaceToDepth_1.md (hunk 3 above)
Comment (pelszkow) [reply docs/ops/movement/SpaceToDepth_1.md:24]: done

File: docs/ops/movement/SpaceToDepth_1.md (hunk 4 above)
Comment (pelszkow) [reply docs/ops/movement/SpaceToDepth_1.md]: done

**Pull request merged**